src/routes/transactions/entities/transfers/erc721-transfer.entity.ts

```diff
@@ -5,8 +5,8 @@ import {
 } from '@/routes/transactions/entities/transfers/transfer.entity';
 
 export class Erc721Transfer extends Transfer {
-  @ApiProperty()
-  tokenAddress: string;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  tokenAddress: `0x${string}` | null;
```

Comment:
Given that it's a required field we should keep it that way on the API level.

Reply:
Apologies, missed that! Reverted in 9bdf73d6951ac2ba52fb2643dbf654b3ff98d65d.